tests/Node.py

```diff
@@ -963,6 +963,41 @@ def getTableColumns(self, contract, scope, table):
         keys=list(row.keys())
         return keys
 
+    def pushTransaction(self, trans, opts="--skip-sign", silentErrors=False, permissions=None):
```

Comment:
Seems odd to have `--skip-sign` as the default.

Reply:
yeah, not sure how I did that.